server/server_test.go

```diff
@@ -249,7 +250,9 @@ func runTestLoadData(c *C) {
 		c.Assert(err, IsNil)
 		err = os.Remove(path)
 		c.Assert(err, IsNil)
+		variable.GoSQLDriverTest = false
 	}()
+	variable.GoSQLDriverTest = true
```

Comment:
Because go-sql-driver regards the warnings as errors.
I add comments in `sessionctx/variable/session.go`.